tsconfig.json

```diff
@@ -37,7 +37,8 @@
     "allowJs": true,
     "checkJs": false,
     "skipLibCheck": true,
-    "typeRoots": ["./node_modules/@types", "./node_modules/@machinomy", "./src/debugger/types"]
+    "typeRoots": ["./node_modules/@types", "./node_modules/@machinomy", "./src/debugger/types", "./typings/"],
+    "types": ["mocha", "chai", "node", "global"]
```

Comment:
this was historic, its probably good to remove.

Reply:
removed this and the typings

Reply:
Nice, we could just leave `mocha`, since the rest is already auto-imported by TS

```json
    "typeRoots": ["./src/debugger/types"],
    "types": ["mocha"],
```